content/ADR-201-lambdas-wearables-API-analysis.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: adr
+adr: 201
+title: Lambdas Wearables API Analysis
+date: 2020-04-03
+status: Draft
+type: RFC
+spdx-license: CC0-1.0
+authors:
+  - pedrotambo
+---
+
+## Abstract
+
+The Explorer uses the Lambdas API `/collections/wearables` to get Wearables. This API sometimes has bad performance, it has an ill design and the handrail between the Explorer and the Content-Server through Lambdas might not be necessary. In this document we review the functionalities of this API and analize solutions to this problems. 
+
+## Context, Reach & Prioritization
+
+During the last MVFW, a report of bad performance APIs was done. Among those APIs, there was the Lambdas API `GET /collections/wearables`.
+
+### API Funcionality
+First, a review of the functionalities of the API will be covered. The API, based on the request filters, gets the wearables from the content server, applies a small transform and returns the result to the client.
+
+#### Request filters
+The API allows three query parameters:
+
+- `wearableIds`: A list of wearable URNs that the client requests. Just ask for them to the content server and apply the transform. Sometimes many wearables are requested that the url string exceeds the maximum allowed.
```

Comment:
I'd replace _Just ask for them to the content server_ for **Just ask the content server for them.**